pkg/services/user/model.go

```diff
@@ -280,9 +280,3 @@ type AdminCreateUserResponse struct {
 	ID      int64  `json:"id"`
 	Message string `json:"message"`
 }
-
-type Password string
-
-func (p Password) IsWeak() bool {
```

Comment:
https://github.com/grafana/grafana/pull/82268#discussion_r1489125227

Lino can you take a look at this?